adapter_pipelines/ss2_single_sample/dependencies.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this dependencies file only used by the ipython notebook?

Reply:
I imagine that lira will need to be told which wdls to import, so I think it will be used by lira as well.

Reply:
Refer to https://github.com/HumanCellAtlas/lira/blob/2362b238d706cfec2d53a26de6bbbbf4945aa8f3/lira/test/data/config.json#L21, I assume if we update the config file of lira in that way, we may don't need this dependenceis.json anymore?

Reply:
Oh, I see now! Previously the listener was given a path to a zip file with all of the dependencies, but the latest listener is getting all of the files from github so this makes sense

Reply:
Thanks for pointing that out @rexwangcc! I think it makes sense to include links to the dependency files in the config directly since that is where the links to the wdl and its input files are also defined.

Right now the listener is expecting a list of analysis wdls in the config file https://github.com/HumanCellAtlas/lira/blob/master/lira/api/notifications.py#L45 so I think we can remove this file @ambrosejcarr

Reply:
Alright, if that's clear. Might support keeping it in master so things are clear, but agree that it's not needed here. :+1: